src/github3/pulls.py

```diff
@@ -958,7 +958,9 @@ def _update_attributes(self, review):
         self.body = review["body"]
         self.body_html = review["body_html"]
         self.body_text = review["body_text"]
-        self.commit_id = review["commit_id"]
+        # NOTE(pabelanger): In some cases, commit_id could be missing on a
+        # PullReview.
```

Comment:
We document cases where it hash items can be `None` in the docstring for the class, e.g. https://github.com/sigmavirus24/github3.py/blob/master/src/github3/pulls.py#L698-L703